src/main/java/seedu/address/logic/commands/people/PeopleReceivedCommand.java

```diff
@@ -24,28 +25,26 @@
 import seedu.address.model.transaction.TransactionList;
 
 /**
- * Records that the person, who owe the user money has paid the user back.
+ * Records that the user has received the amount money lent to a person,
+ * specified by an index in the address book.
  */
-public class PeoplePaidCommand extends Command {
+public class PeopleReceivedCommand extends Command {
 
-    public static final String COMMAND_WORD = "paid";
+    public static final String COMMAND_WORD = "received";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Records that the person has paid you back.\n"
-            + "Parameters: <person's index> (must be a positive integer) "
-            + "[<loans's index> (must be a positive integer)]\n"
+            + "Parameters: <person's index> (must be a positive integer) [" + PREFIX_TRANSACTION_INDEX
+            + "<loans's index> (must be a positive integer)]\n"
             + "Example: " + PEOPLE_COMMAND_TYPE + " "
             + COMMAND_WORD + " 4 1";
```

Comment:
Example usage should have the i/

Reply:
the PREFIX_TRANSACTION_INDEX is the i/

Reply:
i think he mean that ur example got " 4 1"